user-lock-manager-w-keypad-support.smartapp.groovy

```diff
@@ -69,7 +72,7 @@ def setupPage() {
             resetCodeUsage(user)
           }
           if (settings."userCode${user}" && settings."userSlot${user}") {
-            getConflicts(settings."userSlot${user}")
+            //getConflicts(settings."userSlot${user}")
```

Comment:
I had to comment out the calls to getConflicts due to it preventing me from accessing the user settings in a second instance of the SmartApp. It was giving me a 'you are not authorized...' error message.